Assets/LeapMotionModules/DetectionExamples/Scripts/LeapRTS.cs

```diff
@@ -57,39 +37,49 @@ public enum RotationMethod {
     private float _defaultNearClip;
 
     void Start() {
-//      if (_pinchDetectorA == null || _pinchDetectorB == null) {
-//        Debug.LogWarning("Both Pinch Detectors of the LeapRTS component must be assigned. This component has been disabled.");
-//        enabled = false;
-//      }
-
       GameObject pinchControl = new GameObject("RTS Anchor");
       _anchor = pinchControl.transform;
       _anchor.transform.parent = transform.parent;
       transform.parent = _anchor;
     }
 
+    private Hand _lHand;
```

Comment:
These Hand vars dont seem to need to be defined as fields, they could be local vars to Update right?